docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md

```diff
@@ -5,7 +5,7 @@ description: 'Architecture documentation (arc42)'
 sidebar_position: 1
 ---
 
-![Business partner kit banner](@site/static/img/BPKitIcon.png)
+![Business partner kit banner](/img/kit-icons/bp-kit-icon.svg)
```

Comment:
Is that correct that you reference this directly?

And what happend to the BPKitIcon.png file?

Reply:
> Is that correct that you reference this directly?
> 
> And what happend to the BPKitIcon.png file?

> @Siegfriedk  Yes, It is correct. 
>The changes in format will be take effect in the next version. I have created a new folder named "kit-icons" where all the icon images are stored in SVG format.

Reply:
Just to be clear right: You are aware that docusourus uses @site to determine a relative path if the page would deloyed differently and you use the static path without any @site fully aware what it means?

Reply:
Hello @Siegfriedk, if you refer [Docusaurus Docs](https://docusaurus.io/docs/static-assets#in-markdown) on usage of static assets under markdown file. It is clearly mentioned the image references should not use `@site`, but just need to use absolute paths and we checked rendering is not being affected even if we provide static path.